src/awkward/types/listtype.py

```diff
@@ -47,7 +47,7 @@ def _str(self, indent, compact):
             out = ["string"]
 
         elif self.parameter("__array__") == "bytestring":
-            out = ["bytes"]
+            out = ["byte"]
```

Comment:
Same: it has to be `"byte"` to work, and the convention has already been out in the community (in principle). Even with a major release, it would be too messy to change.

Also, the singular is technically right, because the type name describes an element of the array (singular), not the array (plural).